clientSide/src/components/radioButton/RadioButton.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./RadioButton.css";
+
+const RadioButton = (props) =>{
+    return(
+        <div className="RadioButton">
```

Comment:
Why is this className PascalCase when all others are kebab-case. Be consistent. It is important.